editor/scene/canvas_item_editor_plugin.cpp

```diff
@@ -788,6 +812,18 @@ void CanvasItemEditor::_find_canvas_items_in_rect(const Rect2 &p_rect, Node *p_n
 				r_items->push_back(ci);
 			}
 		}
+
+		// gizmos way
+		Vector<Ref<CanvasItemGizmo>> gizmos = ci->get_gizmos();
+		for (int i = 0; i < gizmos.size(); i++) {
+			Ref<EditorCanvasItemGizmo> gizmo = gizmos[i];
```

Comment:
```suggestion
		for (Ref<EditorCanvasItemGizmo> gizmo : gizmos) {
```
Will handle the casting